app/components/mandatarissen/confirm-bekrachtiging.hbs

```diff
@@ -0,0 +1,25 @@
+<AuModal
+  @title={{(concat "Bekrachtig " @mandataris.bekleedt.bestuursfunctie.label)}}
+  @modalOpen={{@modalOpen}}
+  @closeModal={{@cancel}}
+  as |Modal|
+>
+  <Modal.Body>
+    <p>
+      Bent u zeker dat u het mandaat wil bekrachtigen? Deze actie is niet
```

Comment:
Add something about this action happening automatically based on the decisions published as a result of the gemeenteraad, but that it can be done manually if necessary?